app/spaces/[spaceid]/trustful/share/components/ShareSection.tsx

```diff
@@ -0,0 +1,131 @@
+'use client';
+
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useEffect } from 'react';
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Text,
+  Flex,
+  Link,
+} from '@chakra-ui/react';
+import { useParams, useRouter } from 'next/navigation';
+import { BeatLoader } from 'react-spinners';
+import { useAccount } from 'wagmi';
+import toast from 'react-hot-toast';
+import { getEllipsedAddress } from '@/utils/format';
+import { BlockfulLogo, QrCodeIcon } from '@/components/icons';
+import { QRCodeGiveBadge } from '@/app/spaces/[spaceid]/trustful/share/components';
+import {
+  CopyToClipboardButton,
+  TheFooterNavbar,
+  TheHeader,
+} from '@/app/spaces/[spaceid]/trustful/components';
+
+export const ShareSection = () => {
+  const { address, chain } = useAccount();
+  const { push } = useRouter();
+  const params = useParams();
+
+  const actualURL = `/spaces/${params.spaceid}/trustful`;
+
+  // const { villagerAttestationCount } = useContext(WalletContext);
+  const villagerAttestationCount = Number(1);
```

Comment:
Please review this definition

Reply:
Yes. It's under development.

Reply:
Can you please add TODO comments where you find space of open action items?